src/Level02/Core.hs

```diff
@@ -54,8 +54,7 @@ resp400
 resp400 =
   error "resp400 not implemented"
 
--- These next few functions will take raw request information and construct one
--- of our types.
+-- | These next few functions will take raw request information and construct one of our types.
```

Comment:
This seems wrong. This attaches the text to one specific function. There are several of these comment blocks.